doc/release-notes/release-4.sgml

```diff
@@ -209,6 +209,10 @@ This section gives a thorough account of those changes in three categories:
 	<p>New directive to limit the size of a table used for sharing information
 	   about collapsible entries among SMP workers.
 
+	<tag>hopeless_kid_revival_delay</tag>
+	<p>New directive to set a cool-down delay reviving a child process if
```

Comment:
I suggest documenting this differently because the new directive changes default Squid behavior and because it was introduced not so much to _delay_ hopeless kids revival but to make such revival possible in the first place (the delay is a side effect). Consider a description like this:

New directive and new behavior. The directive sets the delay between revival attempts of a child process experiencing repeated and frequent deaths. Squid no longer gives up on such processes forever but, instead, tries to revive them an hour after their last hopeless designation.

Reply:
These are one-liner descriptions of just what the config setting does. The cf.data.pre has full description with behaviour context for admin using it.

As such it is paraphrasing "A kid experiencing frequent deaths is marked as "hopeless" for the duration specified by this directive", without the custom terminology.

If the meaning of the original sentence is different to what it appears to be saying then please PR an update to fix both sets of text ASAP so the backport of the main patch can go ahead. It would be better for you to write anyway since it is your fix.

Reply:
> These are one-liner descriptions of just what the config setting does.

I disagree that release notes should limit themselves to one-liners describing a new control knob for a new feature. IMO, the notes should _also_ describe significant changes (to the old Squid behavior), especially when those changes are enabled by default. The wording I proposed does both IMO.

Needless to say, squid.conf.documented is the wrong place to describe changes to old Squid version behavior. Thus, the existing hopeless_kid_revival_delay description in cf.data.pre seems fine to me -- it correctly describes the option value effect/meaning. The functionality _changes_ (compared to the old Squid) are not limited to that effect though -- we did more than making an old constant configurable.